csv-diff: add page (#8520)
tldr-pages/tldr: pages/common/csv-diff.md

# csv-diff

> View differences between two CSV, TSV or JSON files.
> More information: <https://github.com/simonw/csv-diff>.

> - Show a human-readable summary of differences between files, treating the column `key` as an unique identifier:

Reviewer: Perhaps this better
```suggestion
- Show a human-readable summary of differences between files using a specific column as an unique identifier:
```

> `csv-diff {{one.csv}} {{two.csv}} --key={{key}}`

Reviewer: ```suggestion
`csv-diff {{one.csv}} {{two.csv}} --key={{column_name}}`
```

> `csv-diff {{one.csv}} {{two.csv}} --key={{column_name}}`

Reviewer: ```suggestion
`csv-diff {{path/to/file1.csv}} {{path/to/file2.csv}} --key={{column_name}}`
```

- Show a human-readable summary of differences between files that includes unchanges values in rows that have at least one change:

> `csv-diff {{one.csv}} {{two.csv}} --key={{key}} --show-unchanged`

Reviewer: ```suggestion
`csv-diff {{one.csv}} {{two.csv}} --key={{column_name}} --show-unchanged`
```

> `csv-diff {{one.csv}} {{two.csv}} --key={{column_name}} --show-unchanged`

Reviewer: ```suggestion
`csv-diff {{path/to/file1.csv}} {{path/to/file2.csv}} --key={{column_name}} --show-unchanged`
```

> - Show a machine-readable summary of differences between files, treating the column `key` as an unique identifier:

Reviewer: ```suggestion
- Show a summary of differences between files in JSON format using a specific column as an unique identifier:
```

> - Show a summary of differences between files in JSON format using a specific column as an unique identifier:

Reviewer 2: ```suggestion
- Display a summary of differences between files in JSON format using a specific column as a unique identifier:
```

> `csv-diff {{one.csv}} {{two.csv}} --key={{key}} --json`

Reviewer: ```suggestion
`csv-diff {{one.csv}} {{two.csv}} --key={{column_name}} --json`
```

> `csv-diff {{one.csv}} {{two.csv}} --key={{column_name}} --json`

Reviewer: ```suggestion
`csv-diff {{path/to/file1.csv}} {{path/to/file2.csv}} --key={{column_name}} --json`
```
